machine_management/adding-rhel-compute.adoc

```diff
@@ -0,0 +1,39 @@
+[id="adding-rhel-compute"]
+= Adding RHEL compute machines to an {product-title} cluster
+include::modules/common-attributes.adoc[]
+:context: adding-rhel-compute
+
+toc::[]
+
+In {product-title}, you can add Red Hat Enterprise Linux (RHEL) compute, or worker, machines to a user-provisioned infrastructure cluster. You can use RHEL as the operating system on only compute machines.
+
+include::modules/rhel-compute-overview.adoc[leveloffset=+1]
+
+include::modules/rhel-compute-requirements.adoc[leveloffset=+1]
+
+== Preparing an image for your cloud
+
+Amazon Machine Images (AMI) are needed since various image formats cannot be used directly by AWS. You may use the AMIs that Red Hat has provided, or you can manually import your own images. The AMI must exist before the EC2 instance can be provisioned. You will need to list the AMI IDs so that the correct {op-system-base} version needed for the compute machines is selected.
```

Comment:
```
You will need to list the AMI IDs so that the correct {op-system-base} version needed for the compute machines is selected.
```
The previous statement is making a requirement out of listing the AMI IDs. That is not a requirement. The requirement is that you use a valid AMI ID. We can offer listing the AMI IDs as one way to select a valid AMI ID. But we should not prescribe listing the AMI IDs.